repository/c.json

```diff
@@ -733,6 +733,17 @@
 			]
 		},
 		{
+			"name": "Chinese-English Bilingual Dictionary",
+			"details": "https://github.com/divinites/cndict",
+			"previous_names": ["Youdao English-Chinese Translate"],
```

Comment:
You'll want to add `Cndict` as a previous name, otherwise users will get stuck on the version they have installed.